operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -16,6 +16,14 @@ type MultiClusterObservabilitySpec struct {
 	// Advanced configurations for observability
 	// +optional
 	AdvancedConfig *AdvancedConfig `json:"advanced,omitempty"`
+	// Size read path of your Observability instance
+	// +optional
+	ReadTShirtSize TShirtSize `json:"readTShirtSize,omitempty"`
```

Comment:
maybe we can follow suit with Loki etc and just call this `size`?

Reply:
Sure! I kinda wanted to convey that there is a linear scale associated with this

Reply:
I see, the enum values generated for docs should provide enough context imho. They will have to be described anyway in terms of what each size means in terms of infrastructure.

Reply:
Renamed to `InstanceSize` for now